dashboard/package.json

```diff
@@ -50,7 +50,7 @@
     "react-markdown": "^4.2.2",
     "react-minimal-pie-chart": "^6.0.1",
     "react-modal": "^3.1.11",
-    "react-redux": "^5.0.6",
+    "react-redux": "^7.2.0",
```

Comment:
Needed for the redux hooks as per https://react-redux.js.org/api/hooks (well, update to 7.1)